andela/ah-orcas-frontend
# #161254617 reset password

#### What does this PR do?
- Creates the front end interface for reset password.

#### Description of Task to be completed?
- A user should be presented with a button and or link to reset their passwords on the SignUp/Register component.

#### How should this be manually tested?
- git clone this repo
```
$ git clone https://github.com/andela/ah-orcas-frontend/
```
- git checkout to this branch
```
$ git checkout ft-reset-password-161254617
```
- Install the dependencies
```
$ npm install
```
- Start the app
```
- npm start
```
- Check the button on the browser.

#### Any background context you want to provide?
- The tests are still due.

#### What are the relevant pivotal tracker stories?
- [#161254617](https://www.pivotaltracker.com/story/show/161254617)

#### Screenshots (if appropriate)
<img width="718" alt="screenshot 2018-12-06 at 11 30 45" src="https://user-images.githubusercontent.com/26874561/49571910-973cae80-f94b-11e8-9b5e-76a8bdc7f8b8.png">
<img width="1679" alt="screenshot 2018-12-06 at 16 48 14" src="https://user-images.githubusercontent.com/26874561/49632035-56a16b80-fa05-11e8-8e9c-befded0f186c.png">
<img width="1678" alt="screenshot 2018-12-06 at 16 47 42" src="https://user-images.githubusercontent.com/26874561/49632038-5d2fe300-fa05-11e8-807f-1e15f0be410a.png">



#### Checklist:
